src/twisted/topfiles/6800.feature

```diff
@@ -0,0 +1 @@
+twisted.internet.ssl.CertificateOptions now takes a tlsProtocols parameter, which allows you to explicitly pick the versions of TLS or SSL you wish to negotiate. For example, you can now specify TLSv1.2 and TLSv1.0 but not TLSv1.1.
```

Comment:
I don't actually see how to use the API to implement the example given here.  What did I miss?